### ISSUE TITLE
Continuous verification fails to start.

### ISSUE THREAD

**Issue by keith-turner** [apache/accumulo#1695]
While testing 1.10 RC2 the continuous verification script failed to start w/ an error message about bad options.  Working w/ @karthick-rn   we debugged the issue and determined that it may be related to an update in jcommander.

We added set -x to the script and saw the following line was passing `''` for `$SCAN_OPT` and `$AUTH_OPT`.  

https://github.com/apache/accumulo/blob/cfb33e1dcad4ab7d6e07d0b65e20ef9d4d8cf7d5/test/system/continuous/run-verify.sh#L42

After removing `"$SCAN_OPT"` and `"$AUTH_OPT"`  from the line above verify started running.  My suspicion is that the new version of jcommander does not like empty options where the older version ignored them, however I am not certain about this.  Another possibility is that Hadoop tool runner used to drop empty options and now it does not.

**Comment by ctubbsii** [comment]
This could be related to #1684 , which updated jcommander. But, if the opts aren't needed, maybe they shouldn't be there in the first place... or if they are optional, then maybe we should only conditionally add them to the command-line (or simply unquote them in the script, so they don't get passed as empty args when they are empty).

**Comment by karthick-rn** [comment]
@ctubbsii Can you assign this bug to me? Thanks

**Comment by ctubbsii** [comment]
@karthick-rn Any progress on this bug? No rush. Just curious.

**Comment by karthick-rn** [comment]
@ctubbsii Yes, this is on track in-fact I'm currently testing the fix that @keith-turner & I discussed few weeks ago. As we suspected, the issue is related to jcommander update. The previous version of jcommander(v1.48) seem to ignore empty arguments, this behaviour has been changed through https://github.com/cbeust/jcommander/pull/311 and jc no longer ignores empty arguments. I will PR this one soon. 
Also, I'm working on updating `$ACCUMULO_HOME/bin/tool.sh` script to use `zookeeper-*.jar` files from `$ZOOKEEPER_HOME/lib` directory for ZK versions >= 3.5. This is another issue that we ran into whilst testing 1.10 RC2, at the time we manually copied the jar files out of the `/lib` directory. I'll create a separate issue for this and a separate PR. Also, I'll verify the `accumulo-testing` suite as well. Thanks
